src/conn/conn_oligarch.c

```diff
@@ -1008,15 +1078,82 @@ __wt_disagg_put_meta(
     WT_CLEAR(put_args);
     if (disagg->page_log_meta != NULL) {
         WT_ASSERT(session, disagg->bstorage_meta == NULL);
-        return (disagg->page_log_meta->plh_put(
+        WT_RET(disagg->page_log_meta->plh_put(
           disagg->page_log_meta, &session->iface, page_id, checkpoint_id, &put_args, item));
+        __wt_atomic_addv64(&disagg->num_meta_put, 1);
```

Comment:
Is this for statistics?  If so, we should use the regular stat mechanism.

Reply:
It's not used for statistics, but it might be a nice stat to add in the future. This is used internally as a quick way to determine whether any stable tables were checkpointed during the global checkpoint. I don't think we have a precedent for using statistics to make important decisions within WT, so I opted to not use the statistics infrastructure for this.